doc/classes/Particles2D.xml

```diff
@@ -39,8 +41,10 @@
 			How rapidly particles in an emission cycle are emitted. If greater than [code]0[/code], there will be a gap in emissions before the next cycle begins. Default value: [code]0[/code].
 		</member>
 		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps">
+			This setting can be used to set the particle system to render at a fixed FPS. For instance, changing the value to 2 will make the particles render at 2 frames per second. Note this does not slow down the particle system itself.
```

Comment:
"This setting can be used" is unnecessary - these are all settings. Compare with the wording of the existing descriptions. I do like the example, just change the first sentence.

The particle system's frame rate. For instance...